### PR TITLE
[BuildBot] Uplift GPU RT version for Linux CI Process

### PR DESCRIPTION
**Pull request by bb-sycl**
Uplift GPU RT version for Linux to 21.13.19438

### CODE REVIEW

**Comment by bader** [comment]
Do we need this PR or https://github.com/intel/llvm/pull/3531?

**Comment by yanfeng3721** [comment]
Reopen this PR due we have a regression for test usm_badmalloc caused by GPURT version > 21.13.19438

**Review by yanfeng3721 — APPROVED** [review]

**Comment by bader** [comment]
@yanfeng3721, please run `summary:run` tests for GPU drivers update.

File: buildbot/dependency.conf [hunk 1]
```diff
@@ -4,8 +4,7 @@ ocl_cpu_rt_ver=2021.11.3.0.09
 # https://github.com/intel/llvm/releases/download/2021-WW11/win-oclcpuexp-2021.11.3.0.09_rel.zip
 ocl_cpu_rt_ver_win=2021.11.3.0.09
 # Same GPU driver supports Level Zero and OpenCL
-# https://github.com/intel/compute-runtime/releases/tag/21.12.19358
-ocl_gpu_rt_ver=21.12.19358
+ocl_gpu_rt_ver=21.13.19438
```
Comment (bader) [inline buildbot/dependency.conf:7]: Please, add a link to the driver binaries.

**Review by bader — CHANGES_REQUESTED** [review]

File: buildbot/dependency.conf (hunk 1 above)
Comment (yanfeng3721) [reply buildbot/dependency.conf:8]: @bader ,I have add the links, thanks.

**Review by bader — APPROVED** [review]

**Pull request merged**